scripts/exglobal_archive.sh

```diff
@@ -155,10 +155,9 @@ if [[ ${HPSSARCH} = "YES" || ${LOCALARCH} = "YES" ]]; then
     if [[ "${mod}" -eq 0 ]] || [[ "${PDY}${cyc}" -eq "${firstday}" ]]; then SAVEFCSTIC="YES" ; fi
 
 
-    ARCH_LIST="${DATA}/archlist"
-    [[ -d ${ARCH_LIST} ]] && rm -rf "${ARCH_LIST}"
-    mkdir -p "${ARCH_LIST}"
-    cd "${ARCH_LIST}" || exit 2
+    [[ -d ${DATA} ]] && rm -rf "${DATA}"
+    mkdir -p "${DATA}"
+    cd "${DATA}" || exit 2
```

Comment:
The jjob header already creates the directory, so doing that again isn't necessary. It also `cd`s into the directory, but it doesn't hurt to make sure of it here.

Reply:
Done.